scripts/manager.py

```diff
@@ -8,7 +8,7 @@
 import time
 import json
 import stat
-
+import manager_utils as m_utils
```

Comment:
All specific changes to scripts/manager.py would be a feature accessible to all exercises in JdeRobot Academy.
In this case, it is the addition of the Rviz plugin. I guess other projects would also be making similar changes. 

Maybe this should go as a separate feature-issue-pr into JdeRobotAcademy and not part of the pick_place exercise addition PR.

@sixer51 @jmplaza  what are your thoughts on the same?

Reply:
Actually, Siddharth has already created a pull request related to this: https://github.com/trunc8/RoboticsAcademy/commits/issue-801